interpreter/Makefile

```diff
@@ -136,7 +136,7 @@ TESTDIR =	../test/core
 TESTFILES =	$(shell cd $(TESTDIR); ls *.wast; ls [a-z]*/*.wast)
 TESTS =		$(TESTFILES:%.wast=%)
 
-.PHONY:		test debugtest partest
+.PHONY:		test debugtest partest dune-test
```

Comment:
Nit: `dunetest` for consistency.

Can you also add such a target for the dune build itself?

Reply:
Fixed and added.